lisa/tools/ntp.py

```diff
@@ -27,6 +27,10 @@ def can_install(self) -> bool:
 
     def install(self) -> bool:
         posix_os: Posix = cast(Posix, self.node.os)
+        if isinstance(self.node.os, Redhat) and posix_os.release_version >= "8.0.0":
+            raise SkippedException(
```

Comment:
Raise LisaException in tools, the SkippedException is used in test suite layer only.